template/package.json

```diff
@@ -37,13 +39,16 @@
   },
   "dependencies": {
     "amqplib": "0.5.5",
+    "dotenv": "8.2.0",
     "esm": "3.2.25",
+    "joiful": "2.0.0",
     "mikro-orm": "3.4.1",
```

Comment:
Please update package versions to the latest. Mikro-ORM, for example, is at 3.5.0